draft-scurtescu-secevent-event-stream-management-api.md

```diff
@@ -449,9 +449,9 @@ the Event Receiver as a result of the above request:
   "iat": "1493856000",
   "events": [
     "urn:ietf:params:secevent:event-type:core:verification" : {
-      "state": "VGhpcyBpcyBhbiBleGFtcGxlIHN0YXRlIHZhbHVlLgo=",
```

Comment:
There is a small benefit to these trailing commas, if later a new attribute or array element is added then the diff will show only the new item, without the trailing commas it will also show the previous last line as changed because a comma had to be appended at the end.